locales/en-US/experiments.ftl

```diff
@@ -24,7 +24,6 @@ activitystreamContributors11Title = Software Engineer
 activitystreamContributors12Title = Senior UX Designer
 cliqzDescription = Get search results faster. Cliqz shows suggestions right below the URL bar as you type. All while maintaining your privacy.
 cliqzIntroduction = Get search results faster, directly in the browser. Start typing and see suggestions from around the Web in real time, right below the URL bar. And because Cliqz avoids collecting personally identifiable data, or building user profiles, your results are more private.
-cliqzPrivacypreamble = The Cliqz Test Pilot experiment is brought to you through collaboration between Cliqz GmbH, and Mozilla. Cliqz is designed to protect your privacy and avoid collecting sensitive information that can be used to create user profiles. However, anonymizing data like this is difficult, and it may still be possible to identify specific individuals.
```

Comment:
@clouserw this wasn't in the build dir...not sure if it needs to be, but i copy-pastaed the build en-us experiments.ftl to pick up the new changes and this was missing